lhotse/dataset/signal_transforms.py

```diff
@@ -243,13 +243,23 @@ def _forward_single(
                 self.frames_mask_size, _max_tot_mask_frames // num_frame_masks
             )
 
-            for _ in range(num_frame_masks):
-                features = mask_along_axis(
-                    features.unsqueeze(0),
-                    mask_param=max_mask_frames,
-                    mask_value=mean,
-                    axis=1,
-                ).squeeze(0)
+            features = features.unsqueeze(0)
```

Comment:
This code is too hard to follow, I think you need a bit more structure, like a well-documented function.
Also, I think torch.randint might possibly be useful.

Reply:
OK, I will warp these into a function.